lib/workers/repository/updates/flatten.ts

```diff
@@ -42,6 +42,16 @@ export function applyUpdateConfig(input: BranchUpgradeConfig): any {
         .replace(regEx(/^\//), '') // remove leading slash
         .replace(regEx(/\//g), '-') // change slashes to hyphens
         .replace(regEx(/-+/g), '-'); // remove multiple hyphens
+      updateConfig.sourceRepo = parsedSourceUrl.pathname.replace(
+        regEx(/^\//),
+        ''
+      ); // remove leading slash
+      updateConfig.sourceRepoOrg = parsedSourceUrl.pathname
+        .replace(regEx(/^\//), '') // remove leading slash
```

Comment:
Wouldn't the `sourceRepo` also benefit from removing any leading slash?

Reply:
The leading slashes for sourceRepo are removed here: https://github.com/elmaso1620/renovate/blob/564b87da4b28f898219b6e0e50210969d0d18f73/lib/workers/repository/updates/flatten.ts#L48

Reply:
OK, changing my recommendation with a new comment